schemaless/create_uuid_map.py

```diff
@@ -106,6 +106,30 @@ def planning_by_address(self, fk, record, parents, children):
         parents.extend(planning_helper.find_by_address(record['address_norm']))
 
 
+class PlanningMapBlkLotLookupMixin:
+    """Mixin to look up Planning records by Map Block Lot."""
+    def planning_by_mapblklot(self, fk, record, parents, children):
+        """Find Planning parents for `record` that match its map block lot.
+
+        This will not add fks to parents or children if they already
+        share a UUID.
+
+        Args:
+            fk: The PrimaryKey for this record.
+            record: The record dict (from `latest_values`). Expects
+                    `mapblklot` to exist in the dict.
+            parents: A list of fks. List is modified in place.
+            children: A list of fks. List is modified in place.
+        """
+        # TODO: There will be many matches, so we need to filter
```

Comment:
Let's add a github issue for this? I think if they can figure out some of this date stuff (i.e. dbi is after planning, etc), it'd be pretty big for them

Reply:
Created #187